test/demo/utils/notificationsRemote.ts

```diff
@@ -81,3 +82,8 @@ export async function getReceivedEvents(executionTarget: Identity, type: Event['
     });
     return events.filter(event => event.type === type);
 }
+
+export async function getDomElementById(executionTarget: Identity, id: string): Promise<ElementHandle> {
```

Comment:
This doesn't feel like it belongs in this file. `notificationRemote` is really meant to be a mirror of the client api but invoked from a remote window. 

We could make a new file for "DOM Utils" or this could arguably live in the puppeteer util file itself. Up to you where you want it.

Reply:
✅